build.gradle

```diff
@@ -18,8 +18,8 @@ allprojects {
 	group = 'org.openbakery'
 	version = versionNumber
 
-	sourceCompatibility = "1.6"
-	targetCompatibility = "1.6"
+	sourceCompatibility = "1.7"
```

Comment:
@renep Maybe you have some feedback on this. I don't know the exact consequences this change can cause.

Reply:
Java 1.6 ist quite old. I have no problem to go to 1.8 or higher

Reply:
I've set the java version to 1.8 and updated the Readme.